README.md

```diff
@@ -76,37 +75,86 @@ let dog = new Dog("Spot");
 dog.speak(); // Spot speaks! Spot says woof!
 ```
 In `Dog`s `constructor`, `super` is used as a `method`. Whereas, `Dog`'s `speak()`
-method used `super` as an `object`. When the `super` keyword is used as method, it
+method used `super` as an `object`. When the `super` keyword is used as a method, it
 calls the parent `class` `Pet` with the parameters passed to `Dog`. This is to ensure
 that `Dog` is an instance of `Pet`. When `super` is used as an `object` that refers to
-a `Pet` instance, it can calls methods of the parent `class` `Pet` explicitly.
+a `Pet` instance, it can call methods of the parent `class` `Pet` explicitly.
 
 ## Explain When `class` Inheritance Should Be Used in JavaScript
 
-JavaScript can be a bit confusing since it is a dynamic language that uses syntactical
+You probably won't use inheritance often, but as your code bases get larger, you are more
+likely to find a need for it. If you're creating a number of objects that have similar
+features, then creating a generic object type to contain all the shared functionality,
+and inheriting those features in more specialized objects can be convenient and useful.
+
```

Comment:
maybe right after the first paragraph is where the message is delivered - "it is recommended that classes never be extended unless..."

Reply:
Then, as sort of a back up to this paragraph, the following paragraphs can go further. If students don't really get the meaning of the next couple of paragraphs, they'll hopefully just get the message - 'don't extend classes' cause of reasons